### DISCUSSION TITLE
Purchasing an app

### DISCUSSION THREAD

**Discussion by DenisNaida** [fhdm-dev/scoppy#152] [General]
Good afternoon
I can't buy your app on Google Play.
I want to use full functionality, how can I buy it differently?

**Comment by DenisNaida** [comment]
Please help me

**Reply by fhdm-dev** [reply]
Hi,

The premium upgrade is only available via Google Play.

**Reply by DenisNaida** [reply]
I'm ready to pay two prices, but payment via Google Play doesn't work for us

**Comment by Padm59** [comment]
I also want to use the app over subsystem for windows but there is also no way to upgrade it without the play store. I already bought premium on my phone so why does it not work on pc?

**Reply by fhdm-dev** [reply]
Hi,

The app can't be upgraded without the Google Play Store.
